website/content/en/docs/concepts/scheduling.md

```diff
@@ -160,6 +160,9 @@ Take care to ensure the label domains are correct. A well known label like `karp
 | karpenter.k8s.aws/instance-gpu-memory                          | 16384       | [AWS Specific] Number of mebibytes of memory on the GPU                                                                                                         |
 | karpenter.k8s.aws/instance-local-nvme                          | 900         | [AWS Specific] Number of gibibytes of local nvme storage on the instance                                                                                        |
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
Can you update the `preview` docs path so that this change carries forward?

Reply:
Yep, was planning on it. Wanted to see the preview first before I copy it everywhere.